### PR TITLE
Fixed window is not defined error when importing Stories component

### PR DESCRIPTION
**Pull request by IvayloSlavchev**

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-icons": "^4.7.1",
+    "react-router-dom": "^6.8.2",
```
Comment (petya0111) [inline client/package.json]: you can uninstall react-router-dom

File: client/src/app/page.tsx [hunk 2]
```diff
@@ -34,6 +36,10 @@ const Home = () => {
 
         {/* @dev This creates some problems with the responsive so we should make it responsive first */}
         { currentWindowScreen.width > 900 ? <CreatePost /> : null }
+
+        {/* <Routes>
```
Comment (petya0111) [inline client/src/app/page.tsx]: remove the route path here

**Review by petya0111 — COMMENTED** [review]
Perfect react hook window size. You can rebase the changes that Simo has done.

File: client/package.json (hunk 1 above)
Comment (IvayloSlavchev) [reply client/package.json]: Done

File: client/src/app/page.tsx (hunk 2 above)
Comment (IvayloSlavchev) [reply client/src/app/page.tsx]: Done

**Review by petya0111 — COMMENTED** [review]
Please in `/pages` folder make the routing only with the file `/pages/chat.tsx` 
It should be only for routing files. But not with folders and styles files in it.
`ChatBody` is a kind of component and should be in `/components` folder.

File: client/src/pages/chat/ChatBody/ChatBody.tsx
Comment (petya0111) [inline client/src/pages/chat/ChatBody/ChatBody.tsx:1]: `ChatBody` is a kind of component and should be in `/components` folder.

File: client/src/pages/chat/Chat.tsx
Comment (petya0111) [inline client/src/pages/chat/Chat.tsx:1]: Please in `/pages` folder make the routing only with the file `/pages/chat.tsx` 
It should be only for routing files. But not with folders and styles files in it.

**Review by petya0111 — APPROVED** [review]
Good work

**Pull request merged**